frontend/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {Button, Card, CardContent, List,} from "@material-ui/core";
+import styled from "styled-components";
+import CardTitle from "../CardTitle/CardTitle";
+import {VerticalSpace} from "../Shared/VerticalSpace";
+import {Centered} from "../Shared/Centered";
+
+type MenuButtonsProps = {
+    readonly textName: string;
+    readonly onClick: () => void;
+};
+
+type MenuComponentProps = {
+    readonly title: string;
+    readonly menuButtonsList: MenuButtonsProps[];
+};
+
+export const Menu = (props: MenuComponentProps) => (
+    <StartMenuCard>
+        <CardContent>
+            <Centered>
+                <CardTitle title={props.title}/>
+                {MenuButtonsList(props.menuButtonsList)}
+            </Centered>
+        </CardContent>
+    </StartMenuCard>
+);
+
+const StartMenuCard = styled(Card)({
+    display: "block",
+    boxSizing: "border-box",
+    minWidth: "300px",
+    paddingTop: "40px",
+    paddingBottom: "40px",
+});
+
+const MenuButtonsList = (props: MenuButtonsProps[]) => (
```

Comment:
Zdaje mi się, że całe MenuButtons ograniczają nam to co wsadzamy do menu. Zdaje mi się, że to nie musi być okreslane na zewnątrz. Może po prostu wrzucić tam ten buttony i tyle :) ?
Ale jak chcesz tak to zostaw.

Ewentualnie, nie wiem czy to znasz, a warto wiedzieć - children:

```
type MenuComponentProps = {
  readonly title: string;
};

export const Menu: FunctionComponent<MenuComponentProps> = ({
  title,
  children,
}) => (
  <StartMenuCard>
    <CardContent>
      <Centered>
        <CardTitle title={title} />
        {children}
      </Centered>
    </CardContent>
  </StartMenuCard>
);

const StartMenuCard = styled(Card)({
  display: "block",
  boxSizing: "border-box",
  minWidth: "300px",
  paddingTop: "40px",
  paddingBottom: "40px",
});
```

I wtedy możesz np:
```
<Menu title={APP_NAME}>
          <Button variant="contained" color="primary">
            Turnieje
          </Button>
          <VerticalSpace height="1rem" />
          <Button variant="contained" color="primary">
            Zawodnicy
          </Button>
        </Menu>
```